projects/observability/src/shared/components/table/data-cell/log-timestamp/log-timestamp-table-cell-renderer.component.ts

```diff
@@ -0,0 +1,69 @@
+import { ChangeDetectionStrategy, Component, Inject, OnInit } from '@angular/core';
+import { DateCoercer } from '@hypertrace/common';
+import {
+  CoreTableCellParserType,
+  TableCellAlignmentType,
+  TableCellParserBase,
+  TableCellRenderer,
+  TableCellRendererBase,
+  TableColumnConfig,
+  TABLE_CELL_DATA,
+  TABLE_COLUMN_CONFIG,
+  TABLE_COLUMN_INDEX,
+  TABLE_DATA_PARSER,
+  TABLE_ROW_DATA
+} from '@hypertrace/components';
+import { LogEvent } from '@hypertrace/distributed-tracing';
+import { ObservabilityTableCellType } from '../../observability-table-cell-type';
+
+@Component({
+  selector: 'ht-log-timestamp-table-cell-renderer',
```

Comment:
Can we decouple this from logs? This is just a general relative timestamp renderer, right?

Reply:
+1

Reply:
Done that!!